app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -125,5 +129,49 @@ public void add(Message message) {
         return null;
     }
 
+    @Override
+    public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
+        switch (getItemViewType(position)) {
+            case VIEWTYPE_MESSAGE_HEADER:
+                final MessageHeaderParent messageHeaderParent = (MessageHeaderParent) getItem(position);
+                final MessageHeaderParent.MessageHeaderHolder messageHeaderHolder = ((MessageHeaderParent.MessageHeaderHolder) holder);
+
+                if (messageHeaderParent.getMessageType() == MessageType.STREAM_MESSAGE) {
+                    messageHeaderHolder.streamTV.setText(messageHeaderParent.getStream());
```

Comment:
This is very small thing. The abbreviation `TV` here is a bit confusing. Can we change it to `TextView`.